arviz/plots/plot_utils.py

```diff
@@ -627,3 +630,34 @@ def filter_plotters_list(plotters, plot_kind):
         )
         return plotters[:max_plots]
     return plotters
+
+
+def get_plotting_method(plot_name, plot_module, backend):
+    """Returns plotting function for correct backend"""
+    _backend = {"mpl": "matplotlib", "bokeh": "bokeh", "matplotlib": "matplotlib"}
```

Comment:
If we want to be stricter about backend names we can remove these five lines of code